llama_index/vector_stores/google/generativeai/__init__.py

```diff
@@ -1,6 +1,7 @@
-from .base import GoogleVectorStore, google_service_context
+from .base import GoogleVectorStore, google_service_context, set_google_auth_credentials
```

Comment:
Instead of duplicating this function 3 times, we can just import the 1 version from genai_extension.py like this:

from llama_index.vector_stores.google.generativeai.genai_extension import set_config

__all__ = [
  "set_config",
  ...
]

Reply:
thanks for the tip here! done